features/verblijfplaatshistorie.feature

```diff
@@ -0,0 +1,169 @@
+# language: nl
+
+Functionaliteit: Tonen van verblijfplaatsen
+
+  Huidige en vorige verblijfplaatsen van ingeschreven personen kunnen worden opgezocht.
+
+  De gebruiker kan de verblijfplaats raadplegen op een specifieke peildatum (in het verleden).
+  De gebruiker kan de verblijfplaatsen raadplegen over een specifieke periode.
+  Filteren op peildatum of periode wordt gedaan op basis van de datum aanvang adreshouding van de verblijfplaats(en).
+  Op periode gefilterde gegevens tonen alle verblijfplaatsen waar de persoon gedurende de periode heeft verbleven. Ook wanneer de persoon al voor de periode op een verblijfplaats verbleef en binnen de gevraagde periode nog steeds, wordt de verblijfplaats opgenomen in het antwoord. Ook wanneer de persoon tijdens (een deel van) de periode op een verblijfplaats verbleef en na de gevraagde periode nog steeds, wordt de verblijfplaats opgenomen in het antwoord.
+
+  Wanneer de datum aanvang gedeeltelijk onbekend is, wordt voor de filtering aangenomen dat de persoon gedurende de gehele onzekerheidstijd op het adres heeft verbleven. Wanneer van de datum aanvang adreshouding alleen het jaar bekend is, wordt aangenomen dat de persoon het hele jaar op deze verblijfplaats heeft verbleven.
+  Wanneer van de datum aanvang adreshouding alleen het jaar en de maand bekend is, wordt aangenomen dat de persoon de hele maand op deze verblijfplaats heeft verbleven.
+  Wanneer de datum aanvang adreshouding geheel onbekend is, wordt voor de filtering aangenomen dat de persoon daar altijd al woonde.
+
+  Bij een historische verblijfplaats wordt een datumTot opgenomen, die gelijk is aan de datum aanvang adreshouding van de daarop volgende (actuele of historische) verblijfplaats.
+
+  Wanneer in de registratie voor de persoon meer dan één verblijfplaats, actueel (categorie 8) of historisch (categorie 58), bevat met exact dezelfde datum aanvang adreshouding, dan wordt alleen het voorkomen met de meest recente datum ingang geldigheid opgenomen. Dit speelt bijvoorbeeld wanneer er een correctie is uitgevoerd op de verblijfplaats, of wanneer er een adres hernummering is gedaan.
```

Comment:
@fsamwel Datum ingang geldigheid is niet beschikbaar in gemeentelijke autorisatie. Dit moet dus verandert worden naar volgorde in gbav data structuur.